apps/ccd/automation/kustomization.yaml

```diff
@@ -16,8 +16,8 @@ resources:
   - ../ccd-definition-store-api/demo-image-policy.yaml
   - ../ccd-definition-store-api/image-policy.yaml
   - ../ccd-definition-store-api/ithc-image-policy.yaml
-  - ../ccd-ac-int-api-gateway-web/image-repo.yaml
-  - ../ccd-ac-int-api-gateway-web/image-policy.yaml
+  # - ../ccd-ac-int-api-gateway-web/image-repo.yaml
+  # - ../ccd-ac-int-api-gateway-web/image-policy.yaml
```

Comment:
@NickAzureDevops same as above, there are still comments, as expected?

Reply:
> @NickAzureDevops same as above, there are still comments, as expected?

i did it the same like this https://github.com/hmcts/cnp-flux-config/blob/master/apps/ccd/automation/kustomization.yaml as one got commented out too @madjava